lua/weapons/weapon_ghost_ak47.lua

```diff
@@ -1,3 +1,5 @@
+AddCSLuaFile()
```

Comment:
Why adding AddCSLuaFile() to all files instead of using two in the include function?